src/Aspire.Dashboard/Components/Controls/SummaryDetailsView.razor.cs

```diff
@@ -44,5 +90,76 @@ private void HandleToggleOrientation()
         {
             Orientation = Orientation.Horizontal;
         }
+
+        if (RememberOrientation)
+        {
+            await ProtectedLocalStore.SetAsync(GetOrientationStorageKey(), Orientation);
+        }
+
+        if (RememberSize)
+        {
+            var panel1FractionResult = await ProtectedLocalStore.GetAsync<float>(GetSizeStorageKey());
+            if (panel1FractionResult.Success)
+            {
+                SetPanelSizes(panel1FractionResult.Value);
+
+            }
+            else
+            {
+                ResetPanelSizes();
+            }
+        }
+        else
+        {
+            ResetPanelSizes();
+        }
+
+        // The FluentSplitter control will render during the async calls above, but with the wrong values.
+        // We need to force a re-render to get the correct values.
+        StateHasChanged();
+    }
+
+    private async Task HandleSplitterResize(SplitterResizedEventArgs args)
+    {
+        if (!RememberSize)
+        {
+            return;
+        }
+
+        var totalSize = (float)(args.Panel1Size + args.Panel2Size);
+
+        var panel1Fraction = (args.Panel1Size / totalSize);
+
+        SetPanelSizes(panel1Fraction);
+
+        if (RememberSize)
```

Comment:
alwways true, For false we return early.

Reply:
Actually, that's a good catch for a different reason. The early bail out was supposed to be removed, because we want it to always set those panel sizes regardless, but only remember them for subsequent page loads if RememberSize is true.